src/Composer/Satis/Command/BuildCommand.php

```diff
@@ -119,6 +119,19 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $config = $file->read();
         }
 
+        try {
+            $this->check($configFile);
+        } catch (\Exception $e) {
+            if (!$skipErrors) {
+                throw $e;
+            }
+            $output->writeln(sprintf("<error>Skipping Exception '%s'.</error>", $e->getMessage()));
```

Comment:
Better to use warning instead of error I think, since `$skipErrors` basically downgrades them.